outlookmail/Module.md

```diff
@@ -54,6 +56,16 @@ public function main() returns error? {
     mail:Message message = check outlookClient->createMessage(draft);
     log:printInfo(message.toString());
 }
-   
+
 ``` 
-## [You can find more samples here](https://github.com/ballerina-platform/module-ballerinax-microsoft.outlook.mail/tree/main/outlookmail/samples)
\ No newline at end of file
+## Quick reference 
+* Get a message
+ ```ballerina
+   mail:Message message = check outlookClient->getMessage("<Message ID>");
+   ```
+* Send an existing draft
```

Comment:
What about `sending a simple email` example? Sending a draft is something advance (beyond very basic)

Reply:
The quick start section has the complete sample for sending a message, here it is just get the message Id of a draft and send it.